cranelift-codegen/meta/src/isa/x86/recipes.rs

```diff
@@ -2449,6 +2449,18 @@ pub(crate) fn define<'shared>(
             ),
     );
 
+    recipes.add_template_recipe(
+        EncodingRecipeBuilder::new("vconst_opt", f_unary_const, 1)
```

Comment:
May I suggest to use a different name for this, please? "opt" is ambiguous in that it could mean either "optional" or "optimized". (And both would be a valid interpretation here. :))